float64 repeat
float64 exposure
float64 stime
string day